Cargo.toml

```diff
@@ -73,6 +73,8 @@ relative-path = { version = "1.7", features = ["serde"] }
 reqwest = { version = "0.11.27", default-features = false, features = [
```

Comment:
would be nice to have the latest version 0.12.x if there is no blocker issue.

Reply:
We will take this up in #1075

Reply:
`actix_http:latest` and `reqwest:v0.11+` are incompatible because one depends on different versions of `http`([ref1](https://github.com/actix/actix-web/blob/master/actix-http/Cargo.toml#L116), [ref2](https://github.com/seanmonstar/reqwest/blob/master/Cargo.toml#L103)).